pkg/frontend/querymiddleware/astmapper/parallel.go

```diff
@@ -27,6 +27,8 @@ var NonParallelFuncs = []string{
 	// The following functions are not safe to parallelize.
 	"absent",
 	"absent_over_time",
+	// TODO: Find out whether should be parallelizable.
+	"info",
```

Comment:
Let's make clear that the comment applies only to this function, not the rest of the functions below.

```suggestion
	"info", // TODO: Find out whether can be parallelizable.
```